contentreview-impl/impl/src/java/org/sakaiproject/turnitin/util/TurnitinLTIUtil.java

```diff
@@ -73,17 +74,21 @@ public void init() {
 		if(ltiService == null){
 			log.warn("TurnitinLTIUtil: Could not find LTI service.");
 		}
-		
-		said = serverConfigurationService.getString("turnitin.aid");
-		secret = serverConfigurationService.getString("turnitin.secretKey");
-		if(said == null || secret == null){
-			log.warn("TurnitinLTIUtil: TII basic configuration is not set.");
-		}			
-		endpoint = serverConfigurationService.getString("turnitin.ltiURL", "https://sandbox.turnitin.com/api/lti/1p0/");		
 		turnitinSite = serverConfigurationService.getString("turnitin.lti.site", "!turnitin");
+		if(StringUtils.isEmpty(turnitinSite)){
```

Comment:
You supply a default in your call to the `ServerConfigurationService` so it should always be defined. The only way this would ever fire is if someone explicitly set `turnitin.lti.site=` which seems to be quite a edge case.